editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
Fair point. I've always found the examples they chose in their documentation to be overly daunting for the sake of showcasing a complex example, but what you've shown here is much simpler.

Another worry is that they don't really prescribe easy patterns for accepting arguments to a selector:

https://github.com/reactjs/reselect#q-how-do-i-create-a-selector-that-takes-an-argument

Here worked out easy enough because `getBlocks` doesn't accept any anyways.